web/app/components/document/index.js

```diff
@@ -1,48 +0,0 @@
-import Component from "@glimmer/component";
-import { inject as service } from "@ember/service";
-import { action } from "@ember/object";
-import { task } from "ember-concurrency";
-
-export default class DocumentIndexComponent extends Component {
-  @service authenticatedUser;
-  @service("fetch") fetchSvc;
-  @service router;
-  @service flashMessages;
-
-  @task *deleteDraft(docID) {
-    // Returns a promise that always results after the provided number of milliseconds
-    const wait = (ms) => new Promise((res) => setTimeout(res, ms));
-
-    try {
-      const docResp = yield this.fetchSvc
-        .fetch("/api/v1/drafts/" + docID, {
-          method: "DELETE",
-          headers: { "Content-Type": "application/json" },
-        })
-        .then((resp) => resp.json())
-        .catch((err) => {
-          console.log(`Error deleting document draft: ${err}`);
-          throw err;
-        });
-
-      // Wait for document to be deleted.
-      yield wait(2000);
```

Comment:
I don't think we need to wait for this since we're redirecting to a known route (unlike the [new doc form](https://github.com/hashicorp-forge/hermes/blob/main/web/app/components/new/doc-form.ts#L219)).